letsencrypt/plugins/manual.py

```diff
@@ -26,13 +26,17 @@
 class Authenticator(common.Plugin):
     """Manual Authenticator.
 
-    .. todo:: Support for `~.challenges.DVSNI`.
```

Comment:
That's a good reason to decouple in general, yes. However, since Manual is not displayed in the UI, it doesn't matter much in this case.